haxepunk/Entity.hx

```diff
@@ -659,12 +659,29 @@ class Entity extends Tweener
 		this.originY = originY;
 	}
 
+	#if html5 
+	static function getInt( value:Dynamic, defaultValue:Int = 0 ): Int
+	{
+		return if (Std.is(value, Int) || Std.is(value, Float)) 
+			value;
+		else 
+			defaulValue;
```

Comment:
This needs to be fixed before merging.

Reply:
Is this a style problem? The Haxe compiler didn't break on me online... I'll fix it!

Reply:
Ah. It was a typo, I see.